server/src/main.rs

```diff
@@ -204,21 +187,25 @@ mod tests {
         assert_test_server_responds_with!(
             "/repo/idontexist?file=README.md&reference=origin/master",
             404,
-            ""
+            "No repo found with name 'idontexist'"
         )
     }
 
     #[test]
     fn get_repo_with_empty_name() {
-        assert_test_server_responds_with!("/repo/?file=README.md&reference=origin/master", 404, "")
+        assert_test_server_responds_with!(
+            "/repo/?file=README.md&reference=origin/master",
+            404,
+            ""
+        )
     }
 
     #[test]
     fn get_repo_with_invalid_file() {
         assert_test_server_responds_with!(
             "/repo/fixtures?file=not-a-file&reference=origin/master",
             404,
-            ""
+            "the path 'not-a-file' does not exist in the given tree; class=Tree (14); code=NotFound (-3)"
```

Comment:
I would like to fix these next two error messages to not be directly `git2`'s wording — but job for another PR! 🤷‍♂